homework-selects/sql_questions.sql

```diff
@@ -1,57 +1,751 @@
 -- Write a query to display: 
 -- 1. the first name, last name, salary, and job grade for all employees.
-SELECT first_name, last_name, salary, job_title
-FROM employees
-         LEFT JOIN jobs USING (job_id);
+SELECT
+    first_name, last_name, salary, job_title
+FROM
+    employees
+LEFT JOIN
+        jobs
+        USING (job_id);
+
+SELECT
+       first_name, last_name, salary, job_title
+FROM
+     employees
+JOIN
+    jobs
+    USING (job_id);
+
+SELECT
+       first_name, last_name, salary, j.job_title
+FROM
+     employees e
+JOIN
+    jobs j
+    ON (j.job_id = e.job_id);
 -- 2. the first and last name, department, city, and state province for each employee.
--- 3. the first name, last name, department number and department name, for all employees for departments 80 or 40.
--- 4. those employees who contain a letter z to their first name and also display their last name, department, city, and state province.
+SELECT
+       first_name, last_name, department_name, city, state_province
+FROM
+     employees
+LEFT JOIN
+         departments
```

Comment:
try to avoid double tabs if isn't required. Just use tabs, no spaces.
 If you want to use construction `USING` it will be better to keep.
```
LEFT JOIN
    table_name USING (common_field)
```

Reply:
Ok,fixed!